frontend/src/components/Forms/variances/VarianceSearchForm.js

```diff
@@ -0,0 +1,162 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { Field, reduxForm } from "redux-form";
+import { Form, Button, Col, Row, Icon } from "antd";
+import * as FORM from "@/constants/forms";
+import { renderConfig } from "@/components/common/config";
+import CustomPropTypes from "@/customPropTypes";
+
+const propTypes = {
+  handleSubmit: PropTypes.func.isRequired,
+  handleVarianceSearch: PropTypes.func.isRequired,
+  toggleAdvancedSearch: PropTypes.func.isRequired,
+  reset: PropTypes.func.isRequired,
+  isAdvanceSearch: PropTypes.bool,
+  complianceCodes: CustomPropTypes.options.isRequired,
+  mineRegionOptions: CustomPropTypes.options.isRequired,
+};
+
+const defaultProps = {
+  isAdvanceSearch: false,
+};
+
+const validate = (values) => {
+  const errors = {};
+  if (values.issue_date_min && values.issue_date_max) {
+    if (Date.parse(values.issue_date_min) > Date.parse(values.issue_date_max)) {
+      errors.issue_date_max = "Must be after issue date.";
+    }
+  }
+  if (values.expiry_date_min && values.expiry_date_max) {
+    if (Date.parse(values.expiry_date_min) > Date.parse(values.expiry_date_max)) {
+      errors.issue_date_max = "Must be after expiry date.";
+    }
+  }
+  return errors;
+};
+
+export class VarianceSearchForm extends Component {
+  handleReset = () => {
+    this.props.reset();
+    this.props.handleVarianceSearch({}, true);
+  };
+
+  render() {
+    return (
+      <Form layout="vertical" onSubmit={this.props.handleSubmit} onReset={this.handleReset}>
+        <Row gutter={6}>
+          <Col md={24} xs={24}>
+            <Field
+              id="search"
+              name="search"
+              component={renderConfig.FIELD}
+              placeholder="Search by code section, mine name, or mine number"
+            />
+          </Col>
+        </Row>
+        {this.props.isAdvanceSearch && (
+          <div>
+            <Row gutter={6}>
+              <Col md={24} xs={24}>
+                <Field
+                  id="compliance_code"
+                  name="compliance_code"
+                  placeholder="Select Compliance Code"
+                  component={renderConfig.MULTI_SELECT}
+                  data={this.props.complianceCodes}
+                />
+              </Col>
+            </Row>
+            <Row gutter={6}>
+              <Col md={12} xs={24}>
+                <Field
+                  id="issue_date_min"
+                  name="issue_date_min"
+                  placeholder="Select Min Issue Date"
```

Comment:
Is "min" the "earliest" date to be included in the range of valid dates? Such that anything before this date is filtered out?

If so, something like "earliest" might be more intuitive than "min," but I'd suggest confirming this with UX if you haven't already